src/orgasm_control.c

```diff
@@ -343,10 +346,17 @@ static void orgasm_control_updateEdgingTime() { // Edging+Orgasm timer
         // Detect if within post orgasm session
         if ((esp_timer_get_time() / 1000UL) < (post_orgasm_state.post_orgasm_start_millis +
                                                post_orgasm_state.post_orgasm_duration_millis)) {
-            output_state.motor_speed = Config.motor_max_speed;
+            // continue to raise motor to max speed
+            if (output_state.motor_speed <= (Config.motor_max_speed - output_state.motor_increment)
+                ) {
+                update_check(output_state.motor_speed, output_state.motor_speed + output_state.motor_increment);
+            } else {
+                update_check(output_state.motor_speed, output_state.motor_speed + output_state.motor_increment);
+            }
```

Comment:
I tried using _set_speed here and display did not render correctly

Reply:
_set_speed controls hardware, but does not trigger an update check. it's a mess.